lerobot/scripts/train.py

```diff
@@ -434,11 +462,22 @@ def evaluate_and_checkpoint_if_needed(step):
 
 @hydra.main(version_base="1.2", config_name="default", config_path="../configs")
 def train_cli(cfg: dict):
-    train(
-        cfg,
-        out_dir=hydra.core.hydra_config.HydraConfig.get().run.dir,
-        job_name=hydra.core.hydra_config.HydraConfig.get().job.name,
-    )
+    if "ACCELERATE_MIXED_PRECISION" in os.environ:
```

Comment:
I have another comment above asking about this env variable. Blocking here too till that's resolved.